lms/templates/instructor/instructor_dashboard_2/instructor_dashboard_2.html

```diff
@@ -116,11 +116,12 @@
         ## to keep this short, sections can be pulled out into their own files
 
         % for section_data in sections:
-          <section id="${ section_data['section_key'] }" class="idash-section">
-            <%include file="${ section_data['section_key'] }.html" args="section_data=section_data" />
+          <section id="${ section_data['section_key'] }" class="idash-section" aria-labelledby="header-${section_data['section_key']}">
```

Comment:
Hm, good catch. I've done so much back and forth with these files, it looks like the `header-` prefix never made it back in. Yes, the `aria-labelledby` here should match the `id` of the child heading. I had it set up to programmatically make this happen, but in all the shuffle, it looks like it was lost. I will get this back in there.

FWIW, the whole instructor dashboard is a tangle and should probably be re-built at some point. Originally I had modified way more files than I needed to, then simplified it to just the base six or so files that are included, then realized that certain settings differ from localhost and a sandbox (e.g., Analytics, etc.) so needed to go through them manually using the `Sections` definition in the main Python loader file. There's gotta be a simpler way to do all of this.